docker/harmony/fi1/pmode.xml

```diff
@@ -95,14 +109,8 @@
         <responderParty name="fi2"/>
       </responderParties>
       <legs>
-        <leg name="testLeg1"/>
-        <leg name="pingServiceCase"/>
```

Comment:
Ping service might still be useful. Was there some specific problem related to that or was this just collateral damage?

Reply:
I removed that because I thought it would be better to ping using tc1Action because then the pmode configuration would be the same in ping than in the actual messages